Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -4122,41 +4140,69 @@ def prettify_applications_arr(applications_arr: Union[List[dict], dict]):
         applications_arr = [applications_arr]
     for i in range(len(applications_arr)):
         application = applications_arr[i]
+        application_characteristics_list=[]
+        for key,value in application.items():
+            if key in CHARACTERISTICS_LIST and value=='yes':
+                application_characteristics_list.append(str(key))
         pretty_application_arr.append({
+            'Category': application.get('category'),
             'SubCategory': application.get('subcategory'),
             'Risk': application.get('risk'),
             'Technology': application.get('technology'),
             'Name': application.get('@name'),
             'Description': application.get('description'),
+            'Characteristics': application_characteristics_list,
             'Id': application.get('@id'),
         })
     return pretty_application_arr
 
 
 @logger
-def panorama_list_applications(predefined: bool) -> Union[List[dict], dict]:
+def panorama_list_applications(args:Dict[str, str], predefined: bool) -> Union[List[dict], dict]:
```

Comment:
```suggestion
def panorama_list_applications(args:Dict[str, str], predefined: bool) -> Union[List[dict], dict]:
```

why do you pass in predefined and also args? pass only args and extract predefined from it